src/crypto/SubtleCryptoProvider.ts

```diff
@@ -18,15 +20,15 @@ class SubtleCryptoProvider extends CryptoProvider {
   }
 
   /** @override */
-  computeHMACSignature(payload, secret) {
+  computeHMACSignature(payload: string, secret: string): string {
     throw new Error(
       'SubtleCryptoProvider cannot be used in a synchronous context.'
     );
   }
 
   /** @override */
   async computeHMACSignatureAsync(payload, secret) {
-    const encoder = new TextEncoder('utf-8');
+    const encoder = new TextEncoder();
```

Comment:
It seems that `TextEncoder` does not take any parameters, and by default the newly created `TextEncoder` object will use UTF-8 encoding. https://developer.mozilla.org/en-US/docs/Web/API/TextEncoder/TextEncoder